lib/menubar/init.lua

```diff
@@ -61,7 +61,7 @@ menubar.geometry = { width = nil,
                      y = nil }
 
 --- Width of blank space left in the right side.
-menubar.right_margin = 50
+menubar.right_margin = theme.xresources.apply_dpi(8)
```

Comment:
Why change the default for this (assuming 96) DPI from 50 to 8? That seems random. I'd like to see at least some reason in the commit message.